.github/workflows/release-publish.yml

```diff
@@ -130,7 +151,7 @@ jobs:
           GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
       -
         name: Publish Krew
-        if: ${{ env.SKIP_KREW == 'false' }}
+        if: ${{ steps.check-branch.outputs.is_latest == 'true' }}
```

Comment:
I see `check-branch` is a job not a step in this job, so should we use following?
`if: ${{ needs.check-branch.outputs.is_latest == 'true' }}`

Reply:
this is also 
https://github.com/cloudnative-pg/cloudnative-pg/pull/4161/files#diff-c6d47dacf31662f6791d0cf218b1d34ee0ab48348c037819431cdd40e3fc23e4R77